db/migrate/20201028134841_update_user_api_key_scopes.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateUserApiKeyScopes < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
Technically this is an `up` but I'm not sure if it makes a difference anymore in migrations.

Reply:
You're right, thanks. Updated :+1: